### ISSUE TITLE
Option to have Movie titles displayed in a language I understand

### ISSUE THREAD

**Issue by tvebax** [Radarr/Radarr#4045]
**Is your feature request related to a problem? Please describe.**
In my mind, it would make more sense if movie titles followed the system / user language, than the individual movie's origin.

Lately Radarr has started to display movie titles in the language of the movie. While I do see the reasoning, this is not helpful in real life.
People watch movies in languages they do not speak - which might also have alphabets they can't read.

Example:
I do not speak either arabic, spanish, italian, finnish, chinese or icelandic. I can't even read arabic or chinese. But I do sometimes watch movies from these countries.

Having a Radarr overview with movie titles like the below, is seriously not helpful:
הערת שוליים
جدایی نادر از سیمین
درباره الی‎‎
おくりびと
万引き家族
기생충
Zimna wojna
Vals Im Bashir
Tro, håb og kærlighed
Tarde para la ira
Sprängaren

I do see the intention for the multilingual features to be helpful for people who speak other languages than english, but. I highly doubt anyone speaks all of the above ^^ which is now necessary to make sense of titles in Radarr.

**Describe the solution you'd like**
I would like to be able to set *my* language in Radarr and have movie titles adhere to what I can understand - not the movie language, whenever possible. (I am aware that some movies never get an international title, but only have a local one. That's an edge case - and absolutely ok).

Basically, I'd like Radarr movie titles to work how it used to, before all the multi lingual stuff got implemented.

**Describe alternatives you've considered**
Additional features could be to let users tick off languages that should be allowed for movie titles. That way they can choose the set they understand.

<img width="747" alt="Movies_-_Radarr" src="https://user-images.githubusercontent.com/42374813/71996951-6e1cda80-323d-11ea-92e3-266802274ec3.png">

**Comment by Qstick** [comment]
Believe this is a duplicate request to #3497

**Comment by tvebax** [comment]
More or less. A button to disable multilingual features and only use english, would be a sufficient solution here. No need to choose language.

A way to eliminate the regression introduced by multilingual features implementation, is the goal.

**Comment by Qstick** [comment]
Closing as dupe
